tests/test_Admin.py

```diff
@@ -36,6 +36,11 @@ def test_basic_api():
     except KafkaException as e:
         assert e.args[0].code() in (KafkaError._TIMED_OUT, KafkaError._TRANSPORT)
 
+    try:
+        a.list_groups(timeout=0.2)
+    except KafkaException as e:
+        assert e.args[0].code() in (KafkaError._TIMED_OUT, KafkaError._TRANSPORT)
+
```

Comment:
Add an integration test (tests/integration/..) too where you:
 - produce some messages to a topic
 - start two consumers
 - wait for them to consume and commit
 - then call list_groups() both with and without the group name
 - verify group has two members

Reply:
Unfortunately I can't get the integration tests running on my mac, it errors with

```
unable to resolve docker endpoint: context "/Users/xxx/Projects/confluent-kafka-python/tests/docker/docker-compose.yaml" does not exist
```

But the file does exists

Reply:
I think you can just run the integration_test.py script directly by creating a testconf.json file with bootstrap.servers set

Reply:
It works.

```
python tests/integration/integration_test.py --admin tests/integration/testconf.json
Using confluent_kafka module version 1.5.0 (0x1050000)
Using librdkafka version 1.5.0 (0x10500ff)
Testing: ['admin']
Brokers: localhost:9092
Topic prefix: test-topic
============================== Verifying Admin API ==============================
Reached end of test-topic_admin_c63da45f-4f28-4160-9d28-8757ec70147c [0] at offset 1
Reached end of test-topic_admin_c63da45f-4f28-4160-9d28-8757ec70147c [1] at offset 0
Reached end of test-topic_admin_c63da45f-4f28-4160-9d28-8757ec70147c [3] at offset 0
Reached end of test-topic_admin_c63da45f-4f28-4160-9d28-8757ec70147c [4] at offset 0
Reached end of test-topic_admin_c63da45f-4f28-4160-9d28-8757ec70147c [2] at offset 1
EOF reached for all assigned partitions: exiting
Reached end of test-topic_admin_c63da45f-4f28-4160-9d28-8757ec70147c [0] at offset 1
Reached end of test-topic_admin_c63da45f-4f28-4160-9d28-8757ec70147c [1] at offset 0
Reached end of test-topic_admin_c63da45f-4f28-4160-9d28-8757ec70147c [3] at offset 0
Reached end of test-topic_admin_c63da45f-4f28-4160-9d28-8757ec70147c [4] at offset 0
Reached end of test-topic_admin_c63da45f-4f28-4160-9d28-8757ec70147c [2] at offset 1
EOF reached for all assigned partitions: exiting
Topic test-topic_admin_c63da45f-4f28-4160-9d28-8757ec70147c marked for deletion
============================== Done ==============================
```